testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/RefreshTokenTest.java

```diff
@@ -1356,6 +1349,168 @@ public void refreshTokenUserClientMaxLifespanGreaterThanSession() throws Excepti
         }
     }
 
+    @Test
+    public void refreshTokenUserSessionMaxLifespanModifiedAfterTokenExchange() throws Exception {
```

Comment:
```suggestion
    public void refreshTokenUserSessionMaxLifespanModifiedAfterTokenRefresh() throws Exception {
```

Minor nitpick to rename this method as it does not have anything to do with "token exchange" and hence is a little bit misleading.

Few other places in this class of similar naming.

Reply:
Done! I have changed all the methods...